### ISSUE TITLE
Missing json config files when converting model from the Hub

### ISSUE THREAD

**Issue by callmephilip** [xenova/transformers.js#11]
I ran the following locally `python ./scripts/convert.py --model_id openai/whisper-tiny.en --from_hub --quantize --task speech2seq-lm-with-past`

Which worked mostly fine. However, the resulting directory containing converted model had a couple of files missing, namely 
`preprocessor_config.json` and `tokenizer.json`. After grabbing them manually from HF https://huggingface.co/openai/whisper-tiny.en/tree/main, things seem to work fine

**Comment by xenova** [comment]
Ah yes, you are completely right. I had this problem before. Initially, I just added a check for which models have processors: https://github.com/xenova/transformers.js/commit/6078e1cbdcde348ba86bcd76be211ed8496ef318

but I removed it since it was extremely hacky, and would be difficult to maintain.

The other option would be to just copy all .json files: https://github.com/xenova/transformers.js/blob/492c1178bfabe260f37fac1777c5216cfe213f4e/scripts/convert.py#L203

Unless there is a better way? I assume this is a problem with optimum and the transformers' `save_pretrained` function, which seems to ignore the tokenizer.json
